LaunchDarkly.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |ld|
 
   ld.name         = "LaunchDarkly"
-  ld.version      = "4.0.0"
+  ld.version      = "4.1.0"
```

Comment:
What makes this a minor release rather than a patch? It looks like the `deviceModel`/`deviceType` changes are internal, but I might be wrong.